src/app/core/classes/common-utils.ts

```diff
@@ -39,5 +39,9 @@ export class CommonUtils {
 		return 0;
 	}
   
+	capitalizeFirstLetter(text:string) {
+		text = text.toLowerCase();
+		return text.charAt(0).toUpperCase() + text.slice(1);
+	}
 }
   
```

Comment:
GH doesn't detect this as a new line, might be ssome line-ending mixup happening here...

Reply:
Sorry, I am not sure I am following you. :)

Reply:
Look at "files changed" end-of-file-whiteline warning. 
It isn't detecting line 47, most often because Linux/Unix and Windows end-of-lines are mixed in the file.

Reply:
Perhaps you can't pull above changes?

Reply:
No I'm saying GH isn't detecting a linebreak correctly.
Look at "files changed" in github, notice it gives an "end of file whiteline" warning icon. 
This is often because mixed lineendings.

Reply:
![image](https://user-images.githubusercontent.com/7613738/112875483-88eaa180-90c4-11eb-82d6-cac40be7a4f4.png)

I'm just giving you a hint... This is getting awkwardly tedieus to make clear...

Reply:
Exactly...xD